client/web/compose/src/components/PageBlocks/SocialFeedConfigurator.vue

```diff
@@ -2,15 +2,15 @@
   <b-tab :title="$t('socialFeed.label')">
     <fieldset
```

Comment:
Could you turn these fieldsets into b-form-groups with b-inputs?

If there is any other such occurrence feel free to fix it